### ISSUE TITLE
[Bug] Routing issues with nginx on Docker compose

### ISSUE THREAD

**Issue by taniarascia** [taniarascia/laconia#15]
There's an `.htaccess` file that does some interesting routing. In order to get `laconia.dev/user` to work, I made a rule that looks for `username-router` in the URL and redirects to the profile if it exists. I couldn't find any way to get nginx working with this setup, so maybe someone knows a better way to handle those URLs.

```apacheconf
RewriteEngine On
RewriteBase /
RewriteCond %{REQUEST_FILENAME}%  !-d
RewriteCond %{REQUEST_FILENAME}%  !-f

# Do not route static files in the public directory
RewriteRule \.(js|css|svg|gif|jpg|jpeg|png)$ -                 [L]

# Route /edit/$ to /edit?list_id=$ 
RewriteRule ^/?edit/(.*?)/?$                 /edit?list_id=$1  [L]

# Route everything else to index.php in the public directory
# Note: username-router get variable used for public user profiles
RewriteRule ^(.*)$                index.php?username-router=$1 [QSA,L]
```

**Comment by kevin-schmitt** [comment]
Hello,

For that problem. Better solution is to use apache and no nginx and keep power of htaccess files for this project. I will push an another PR.
Edit: #17

**Comment by taniarascia** [comment]
Closed in #17.
